ext/oberon/psa/core/library/psa_crypto.c

```diff
@@ -54,6 +54,14 @@
 #include "tfm_builtin_key_loader.h"
 #endif /* PSA_CRYPTO_DRIVER_TFM_BUILTIN_KEY_LOADER */
 
+#if defined(CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK) && !defined(CONFIG_BUILD_WITH_TFM)
+#include <hw_unique_key.h>
```

Comment:
hw_unique_key is not available on all platforms.

You need to check if it's enabled like this:

https://github.com/nrfconnect/sdk-nrf/pull/14184/files#diff-75204901e8853fa90bfb8eb20bfa7e011fc804867a84b0c3887c30abf7f31326R16

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/trusted_storage/Kconfig#L132:L134
`CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK` selects `CONFIG_HW_UNIQUE_KEY` so this dependency should be covered?